chainer/functions/evaluation/classification_summary.py

```diff
@@ -43,6 +42,7 @@ def check_type_forward(self, in_types):
     def forward(self, inputs):
         xp = cuda.get_array_module(*inputs)
         y, t = inputs
+        t = t.astype('i', copy=False)
```

Comment:
Is this conversion necessary?
I think the actual type for `'i'` is environment-dependent.

Reply:
Yes. Some Numpy functions require int32 on Windows.

Reply:
I see. Can you write a brief comment on this line?

Reply:
OK